app/models/hero.rb

```diff
@@ -0,0 +1,3 @@
+class Hero < ApplicationRecord
+  has_many :base_stats
```

Comment:
Consider adding "inversions". Helpful article: https://www.viget.com/articles/exploring-the-inverse-of-option-on-rails-model-associations